resources/views/dashboard/controllers/files.blade.php

```diff
@@ -303,6 +427,25 @@
                 <tbody>
                     @if($staff->count() > 0)
                         @foreach($staff as $f)
+                            @if($f->row_separator)
+                                <tr>
+                                    @if(Auth::user()->isAbleTo('files'))
+                                        <th class="text-center" colspan="3">{{ $f->name }}</th>
```

Comment:
Just noticed this on another look through. Let's pull this <th> block out before the if statement since it should be shown all the time. Then that'll get rid of the else portion.

Reply:
Sorry, I'm not following your comment. Are you referring to the table row or the table header? I can pull the table row out - that doesn't change, but the table header versus table column are different between the separator rows and the file rows.

Reply:
Done - thank you for pointing this out.